1.12.6 fixes

trainvoi opened the pull request.

roquesystemexe committed:
Add files via upload

roquesystemexe committed:
Update README.md

planetme committed:
cabalment complete

planetme committed:
add fiber prop to PBI

loxoDev committed:
Rename en_pt.lang_UNFINISHED.txt to en_pt.lang

roquesystemexe committed:
Add files via upload

roquesystemexe committed:
Merge branch 'SymmetricDevs:master-ceu' into master-ceu

roquesystemexe committed:
Update README_PT.md

last8kings committed:
Update README.md

bruberu committed:
Merge pull request #777 from RoqueCrafter/master-ceu

README.md in Portuguese

MCTian-mi committed:
update zh_cn

MCTian-mi committed:
Merge branch 'Chinese-localization' of https://github.com/MCTian-mi/Supersymmetry into Chinese-localization

MCTian-mi committed:
fix everything

MCTian-mi committed:
remove dot

planetme committed:
Merge pull request #781 from MCTian-mi/Chinese-localization

Replace hard-coded main menu button texts and update chinese localization

TheScienceDemon committed:
Translate Main Menu [DE]

TheScienceDemon committed:
Add more localisation support to main menu

TheScienceDemon committed:
Make mod counter work again

`#modsactive#` and `#modsloaded#` don't work when used as localised strings

TheScienceDemon committed:
Rename Replay Mod button in main menu [DE]

TheScienceDemon committed:
Make questbook.py deactivate edit mode

TheScienceDemon committed:
Make main.py run questbook.py

TheScienceDemon committed:
Make questbook.py actually deactivate edit mode

bruberu committed:
Merge pull request #783 from TheScienceDemon/questbook-file-fix

Questbook.py now correctly sets edit mode + main.py runs questbook.py

bruberu committed:
Merge pull request #778 from last8kings/last8kings-patch-1

Update README.md

bruberu committed:
Merge pull request #782 from TheScienceDemon/main-menu-localisation

Main menu localisation [DE]

planetme committed:
for Susy-Core#211

trainvoi committed:
fix some stoik

bruberu committed:
fix: remove old weaving recipes

bruberu committed:
Sneak in 0.1.12.5

bruberu committed:
Remove calls to flare stack, smoke stack, and dumper RMs

bruberu committed:
Merge branch 'master-ceu' into cabal-pbi-fluid

bruberu committed:
Merge pull request #784 from SymmetricDevs/cabal-pbi-fluid

Cabal pbi fluid, thread to plate stuff

bruberu committed:
fix: questbook press enter to exit prompt

planetme committed:
Merge pull request #787 from SymmetricDevs/bru-build-fix

Fix buildscript

trainvoi committed:
remove circuit 0

trainvoi committed:
buff latex throughput

you get 4 rubber bar/plate per 1 sulfur now

trainvoi committed:
fix PVB stoik and PVAc recipe for glue

planetme reviewed (commented):
dont merge, I just realized there's a big problem with PVB

trainvoi committed:
readd boiling recipe

trainvoi committed:
restore PVB recipe to normal?

htmlcsjs committed:
fix server resource packs (#794)

TheScienceDemon committed:
Replace SDRP with CraftPresence

TheScienceDemon committed:
Remove leftover files from SDRP

bruberu committed:
Merge pull request #796 from TheScienceDemon/discord-rpc-replacement

Discord rpc replacement

roquesystemexe committed:
Update README.md (#793)

* Update README.md

Just my role as playtester

* Taken the liberty to quickly adapt the README.md

---------

Co-authored-by: mtbo <111296252+loxoDev@users.noreply.github.com>

kubuus committed:
Polish Localization (#799)

* Polsih Localization

* Readme change

Fiery36 committed:
Coffee Time (#797)

Re-added the pyrolyse oven recipes for roasted coffee bean, making coffee now obtainable

trainvoi committed:
fix DTBP distillation recipe

planetme committed:
fix br so2 recipe duration

planetme committed:
fix PVB

planetme committed:
Update AirDistillation.groovy

planetme committed:
buh

planetme committed:
Merge branch 'master-ceu' into pr/789
